usaspending_api/common/experimental_api_flags.py

```diff
@@ -17,4 +25,46 @@ def is_experimental_elasticsearch_api(request: Request) -> bool:
     """
     Returns True or False depending on if the expected_header_value matches what is sent with the request.
     """
-    return request.META.get(EXPERIMENTAL_API_HEADER) == ELASTICSEARCH_HEADER_VALUE
+    use_es = request.META.get(EXPERIMENTAL_API_HEADER) == ELASTICSEARCH_HEADER_VALUE
+    if not use_es:
+        mirror_request_to_elasticsearch(request)
+    else:
+        logger.info(
+            f"Found {EXPERIMENTAL_API_HEADER} request header. "
+            f"Will use Elasticsearch for backing store where implemented"
+        )
+    return use_es
+
+
+def mirror_request_to_elasticsearch(request: Union[HttpRequest, Request]):
+    """ Duplicate request and send-again against this server, with the ES header attached to mirror
+        non-elasticsearch load against elasticsearch for load testing
+    """
+    url = request.build_absolute_uri()
+    data = json.dumps(request.data)
+    headers = {
+        **request.headers,
+        HttpHeaders.parse_header_name(EXPERIMENTAL_API_HEADER): ELASTICSEARCH_HEADER_VALUE,
+    }
+
+    logger.warning(
+        f"Mirroring inbound request with elasticsearch experimental header.\n"
+        f"\tOriginial Request details: {request._request}\n"
+        f"\tMirrored Request details: \n\t\turl = {url}, \n\t\tdata = {data}, \n\t\theaders = {headers}"
```

Comment:
do any of our requests have large response bodies? We don't have infinite log space.

Reply:
Oh, excellent catch @BrianZito. I added that only so I personally could see the data when developing with this. That verbose logging should definitely be gotten rid of.

I think that whole log statement should go. Presence of the experimental header should be enough for use to delineate what requests were generated by the app vs. those that were natural.

Reply:
Follow question then: with no logging statements, what does the test assert?

Reply:
I'd rather keep a shorted version of the log statement, just to be able to differentiate between what is being duplicated by the site and what is dev and/or testers checking something through postman etc..